test/benchmark-crud/compatibility_makefile

```diff
@@ -0,0 +1,10 @@
+# This is the Makefile for the build.sh system
```

Comment:
This one can actually be removed, but the one in benchmark-common-tasks must be there because it has some custom build parameters that must override the ones in core versions that are being built with build.sh.